lib/tools/system-calls.js

```diff
@@ -399,18 +408,22 @@ systemCallMethods.adbExec = async function adbExec (cmd, opts = {}) {
   opts.timeout = opts.timeout || this.adbExecTimeout || DEFAULT_ADB_EXEC_TIMEOUT;
   opts.timeoutCapName = opts.timeoutCapName || 'adbExecTimeout'; // For error message
 
+  const outputFormatStdout = 'stdout';
```

Comment:
I would rather use something like

```
const EXEC_OUTPUT_FORMAT = {
  STDOUT: 'stdout',
  FULL: 'full',
};
```

Reply:
it would probably even make sense to add this constant to `systemCallMethods` namespace, so it would be available from client modules

Reply:
Added EXEC_OUTPUT_FORMAT to systemCallMethods and used it here + in tests